jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/PostComment.java

```diff
@@ -152,4 +153,22 @@ public void putAttribute(String attributeName, String attributeValue) {
     public void putAllAttributes(Map<String, String> attributes) {
         this.attributes.putAll(attributes);
     }
+
+    /**
+     * Get a sign of delete
+     *
+     * @return true if the comment has been deleted
+     */
+    public Boolean getDeleted() {
```

Comment:
@mixas27 what do you think about using `DATE_DELETED` of DateTime type instead of boolean?